GBHFacebookImagePicker/Classes/Controller/GBHPhotoPickerViewController.swift

```diff
@@ -80,6 +85,9 @@ class GBHPhotoPickerViewController: UIViewController {
 
         // Fetch photos if empty
         self.getPhotos()
+
+        // Statusbar
+        self.setNeedsStatusBarAppearanceUpdate()
```

Comment:
I don't think this is useful. Maybe it's only required when you change a status bar propreties when the controller is already visible ?

Reply:
You are right. I just removed these lines